js/src/behaviors/tabs.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * Tab control for assessment lens
+ * @param container - Container of tab control component
+ */
+/**
+ * Tabs behaviors
+ */
+export function initTabs(container: HTMLElement = document.body) {
+	updateTabNav(container);
+	initTabNavClickListeners(container);
+	initTabControlClickListeners(container);
+}
+
+function initTabNavClickListeners(container: HTMLElement = document.body) {
+	const buttons = Array.from(container.querySelectorAll('.tab-nav')) as HTMLButtonElement[];
+
+	if (!buttons) {
+		return;
+	}
+
+	for (const button of buttons) {
+		button.addEventListener('click', (e: Event) => {
+			e.preventDefault();
+			const b = e.target as HTMLButtonElement;
+			const direction = b.dataset.nav as string;
```

Comment:
data-nav may be too generating. should probably be data-tab-nav

Reply:
updated